legacy/firmware/ethereum_typed_data.h

```
@@ -31,6 +31,25 @@ static const char *const HIGH_RISK_PRIMARY_TYPES_ORDER[] = {
     "OrderComponents",
 };
 
+/**
+ * keccak256(
+ *      "EIP712Domain(uint256 chainId,address verifyingContract)");
+ */
+static const char DOMAIN_SEPARATOR_TYPEHASH[] = {
+    71,  231, 149, 52,  162, 69,  149, 46,  139, 22,  137,
```

Clang (14.0.6)

[warning] narrowing conversion from constant value 231 (0x000000E7) of type 'int' to signed type 'char' is implementation-defined

(bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions)